backlinks/obs_filter.js

```diff
@@ -0,0 +1,80 @@
+var nest = require('depnest')
+var pull = require('pull-stream')
+var MutantPullReduce = require('mutant-pull-reduce')
+var Value = require('mutant/value')
+
+exports.needs = nest({
+  'sbot.pull.backlinks': 'first'
+})
+
+exports.gives = nest("backlinks.filter.obs", true)
```

Comment:
The naming pattern is `topic.type.module`. As described here: [%G74TIna+iO1ByvYA79vNJyhd7xBiNeuDhLTmaWhBso0=.sha256](https://viewer.scuttlebot.io/%25G74TIna%2BiO1ByvYA79vNJyhd7xBiNeuDhLTmaWhBso0%3D.sha256)

So this should probably be `backlinks.obs.filter`.